source/codegen/metadata/niscope/niscope.proto

```diff
@@ -1700,6 +1701,19 @@ message GetScalingCoefficientsResponse {
   sint32 number_of_coefficient_sets = 3;
 }
 
+message GetStartTimestampInformationRequest {
+  nidevice_grpc.Session vi = 1;
+}
+
+message GetStartTimestampInformationResponse {
+  int32 status = 1;
+  uint32 sys_time_in128_bits_t1 = 2;
```

Comment:
Note: I can add `grpc_name` in the metadata to get these parameter names formatted how we want.